data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`
+			branch navy
+				has "license: Navy"
+			choice
+				`	"No, I'm not with the Navy. You could've just asked without slamming me to the ground."`
+					goto notnavy
+				`	"Is that why you attacked me?"`
+					goto detained
+			label navy
+			choice
+				`	"I am with the Navy, but I didn't come here looking for you."`
+				`	"Is that why you attacked me?"`
+					goto detained
+			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`
+				goto detained
+			label notnavy
+			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
+			label detained
+			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
+			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. Only a few small holes are available for airflow. They lock him in one of the cells, making him the only prisoner in the largely empty building. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
+			`	"Don't know why they bother with that. It's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really. I just got a bit jumpy when I spotted you."`
```

Comment:
Figured while I'm here I should have a go at this.
```suggestion
			`	After the short verdict which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	In the intervening quiet while you wait he looks at you very directly through the cell wall and says, "Don't know why they bother with that. It's not like I hit you very hard." When you don't immediately respond though his eyes drop slightly from meeting yours and he continues, "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really. I just got a bit jumpy when I spotted you."`
```

Reply:
I wanna make the start of the conversation less of a casual banter, to help with Wizrad's point, so I wouldn't go with that last paragraph, it is made even more emotional with the "eyes dropping" part I'd say.

Reply:
Ok, alternative suggestion considering that.
```suggestion
			`	After the short verdict which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	In the intervening quiet while you wait he looks at you very directly through the cell wall and says, "Don't know why they bother with that. It's not like I hit you very hard." His gaze does not waver and when you don't immediately respond he continues, "Perhaps you should receive an apology. I've given up on fighting humanity left and right. You should understand that when I spotted you it was natural to perceive you as a threat."`
```

Reply:
The idea I had was going to trim his dialogue, something like: `While waiting, you look back at the Alpha, to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up, "What is it, are you expecting some sort of apology?"`
Having him seem disinterested and starting off the conversation with a more neutral or annoyed tone.

Reply:
You could go that way, though to me that seems rather hostile and lacking a bridge to the next bit of the conversation you have.

Reply:
Yes it's cut rather abruptly.

@W1zrad which would you say works best for making the start of the conversation with Alitis a bit more in tone with what you had in mind? If you have a suggestion of your own that would work better, could also go with that.

Reply:
Hmm. I do like Arachi's abrasive tone better, but it is hard to transition to the conversation following it. What if we start with that and then just add some extra dialogue to smooth out that transition?
```suggestion
			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default, they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	While waiting, you look back at the Alpha to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up. "What is it, are you expecting some sort of apology? I'd consider my caution to be entirely justified, so you won't get one." After a long pause, he continues, "I'm sure you have questions though. I've given up on fighting humanity left and right, and it seems that I won't have anything better to do for some time. Ask away."`
```
I think this could still be improved, but here's what I'm roughly thinking. I've also made a couple comma edits on the writing where necessary.